_posts/2025-08-22-acoustic-collar.md

```diff
@@ -0,0 +1,74 @@
+
+Famously, wireless communications are very power-hungry. LoRa uses less power by transmitting data through "chirps", at the cost of significantly lower bandwidth and longer uplink times. More importantly, LoRa is still by far the most power-intensive of our system. To reduce this issue, and to avoid blocking the network for too long, it quickly became apparent that we would need to compress our audio files significantly. In fact, if we reduced it enough for a full transmission, we could avoid running inference on-device at all, and classify the sound files with more powerful compute resources elsewhere in the network.  For now, we decided to see how far we could take the compression. And, after implementing SD writes with FATFS, we stumbled upon an interesting--and promising--alternative to audio codecs: autoencoders.
+
+The idea behind autoencoders is as follows: a multi-layer perceptron (the encoder) is trained to compress, for example, an image of a spectrogram into a set of embeddings, which another MLP (the decoder) can then decompress into a reasonably accurate representation of the original image. If you sever the two parts of the model, you get a section which can encode images into a smaller representation (known as the “latent space”) and a section which can decode it to its original state. Why would you use this, instead of compressing your audio algorithmically (e.g., MP3)? In theory, an autoencoder could reduce the amount of information significantly more than an algorithm by ignoring aspects of the audio irrelevant to classification and only focusing on the features that distinguish two spectrograms you care about.
+
+{% include 
+    img_caption.html
+    src="assets/acoustic_species_id/2025-08-22-pre-mel.png"
+    src2="assets/acoustic_species_id/2025-08-22-autoencoder.png"
+    src3="assets/acoustic_species_id/2025-08-22-post-mel.png"
+    caption="Sample autoencoder input and output, spectrograms courtesy of Anu Jajodia"
+%}
+
+Putting everything together, we arrive at a cutting-edge device capable of acting as a low-power edge node within and without an external network: a versatile device for analyzing animal behavioral data, which ultimately has the potential to aid countless conservation efforts. We hope our work will be of use to the San Diego Zoo, and continue to be iterated on for years to come.
+
+This project was made possible thanks to the efforts of **Milo Akerman**, **Francisco Irazaba**, **AnMei Dasbach-Prisk**, and **Hayden Dosseh**, along with the teams who have worked on hardware and software previously. 
```

Comment:
I think the names should be higher at the top. Makes it easier for employers to see the student's credit